f8a_worker/workers/githubstats.py

```diff
@@ -0,0 +1,55 @@
+from bs4 import BeautifulSoup
+import requests
+import logging
+from f8a_worker.base import BaseTask
+
+logger = logging.getLogger(__name__)
```

Comment:
Every task you derive from `BaseTask` has already properly instantiated and tweaked logging. You can accept it by accessing `self.log` - e.g. `self.log.info("Info message")`.

Reply:
updated as per comment